server/embed/etcd.go

```diff
@@ -0,0 +1,860 @@
+// Copyright 2016 The etcd Authors
```

Comment:
I do not understand why do you add this file? It seems it isn't needed in release-3.4.

Reply:
You need to update `embed/etcd.go` instead.

Reply:
Ups, looks like this was added as this file was not present in v3.4 branch.